src/main/java/net/sf/jabref/model/entry/BibEntry.java

```diff
@@ -189,6 +167,20 @@ public void setType(EntryType type) {
     }
 
     /**
+     * Sets this entry's ID, provided the database containing it
```

Comment:
Why is the code moved in this class? Could you keep the old position? I think, the move was on purpose. See https://github.com/JabRef/jabref/pull/1309